parsers/EIA.py

```diff
@@ -415,17 +415,45 @@ def fetch_consumption_forecast(
     )
 
 
+def create_production_storage(
+    fuel_type: str, production_point: Dict[str, float], negative_threshold: float
+) -> Tuple[Optional[ProductionMix], Optional[StorageMix]]:
+    """Create a production mix or a storage mix from a production point
+    handling the special cases of hydro storage and self consumption"""
+    production_value = production_point["value"]
```

Comment:
If we're only using the value from the production point why not require that as an argument instead